packages/aws-client/README.md

```diff
@@ -912,51 +860,38 @@ errors, to allow more specific handling by the caller.
 
 * [SQS](#module_SQS)
```

Comment:
can you explain what changed to rearranged these between static and inner heading?

Reply:
An excellent question - indirectly, I'd assume https://github.com/nasa/cumulus/pull/3518.  It's not clear to me what made this change.  The jsdocs weren't updated with the changed signature in #3518, but a short bit of local testing implies that fixing that doesn't change the behavior from the prior release.